mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = async (req, res) => {
+  try {
+    const deletionResult = await Order.findOneAndDelete({ _id: req.params.orderId });
+    if (!deletionResult) {
+      return res.status(404).json({ error: "Order not found" });
+    }
+    res.json({ message: "Order removed successfully" });
+  } catch (error) {
```

Comment:
The catch block should be more specific about the type of error being caught. It's a good practice to catch specific error types instead of catching the general 'error' type. This can help in handling different types of errors more effectively.